osutil/disks/disks_linux.go

```diff
@@ -145,6 +164,19 @@ var diskFromDeviceName = func(deviceName string) (Disk, error) {
 		major: major,
 		minor: minor,
 	}, nil
+
+}
+
+// diskFromDeviceName is exposed for mocking from other tests via
+// MockDeviceNameDisksToPartitionMapping.
+var diskFromDeviceName = func(deviceName string) (Disk, error) {
```

Comment:
(nitpick^4) maybe move this close to `DiskFromDeviceName` ? 
[I was considering to use knuth arrow notation for this one btw ;)]

Reply:
done